webapp/packages/plugin-connection-custom/src/CustomConnectionPluginBootstrap.ts

```diff
@@ -38,20 +40,39 @@ export class CustomConnectionPluginBootstrap extends Bootstrap {
       getItems: (context, items) => [...items, ACTION_CONNECTION_CUSTOM],
     });
 
+    this.menuService.addCreator({
+      menus: [MENU_NAVIGATION_TREE_CREATE],
+      getItems: (context, items) => [...items, ACTION_CREATE_CONNECTION],
```

Comment:
you need to add isApplicable to prevent adding items that will not be presented in the menu